acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/GitLabPages.scala

```diff
@@ -43,6 +51,10 @@ class GitLabPages(
     def settingsLink(implicit webDriver: WebDriver): WebElement = eventually {
       find(cssSelector("span.nav-item-name.qa-settings-item")) getOrElse fail("Settings link not found")
     }
+
+    def repositoryLink(implicit webDriver: WebDriver): WebElement = eventually {
```

Comment:
Is this method used anywhere?

Reply:
Not anymore. It is a left-over from a different approach to creating a merge request. _Removed_.